docs/install/sandboxes/grpc_bridge.rst

```diff
@@ -22,7 +22,7 @@ Building the Go service
 To build the Go gRPC service run::
 
   $ pwd
-  ~/src/envoy/examples/grpc-bridge
+  envoy/examples/grpc-bridge
```

Comment:
Made these consistent with the other examples. It looks like all of the examples assume that the user knows to `cd` into the appropriate subdirectory.